src/tabs/tab-header-bar.scss

```diff
@@ -71,27 +71,37 @@ $label-horizontal-spacing: awsui.$space-l;
   display: flex;
   max-inline-size: calc(90% - #{$label-horizontal-spacing});
 
-  & > button {
+  & > div,
+  & > div > button {
     background-color: transparent;
+    display: flex;
+    align-items: center;
+    text-align: start;
   }
 }
 
 .tabs-tab-label {
   display: flex;
   align-items: center;
-  padding-block: awsui.$space-scaled-2x-xxs;
-  padding-inline: $label-horizontal-spacing;
   text-align: start;
   position: relative;
 
   @include styles.text-wrapping;
 }
 
 .tabs-tab:not(:last-child) {
-  & > a > .tabs-tab-label,
-  & > button > .tabs-tab-label {
+  & > div {
     margin-inline-end: calc(-1 * #{awsui.$border-divider-section-width});
-    border-inline-end: awsui.$border-divider-section-width solid $separator-color;
+    &:before {
```

Comment:
Why does this border need to move to a pseudo-element?

Reply:
Visually the border was too short / too long so I moved it to the new div I'm using to wrap the tab name + actions.

I was trying to match the original look 1:1 and couldn't do it because the height of the div was too long for the border. I moved it to a pseudo element so I could control the height easier.

If you have any suggestions please let me know